docs/solidity-by-example.rst

```diff
@@ -327,18 +322,15 @@ activate themselves.
             // external contracts.
 
             // 1. Conditions
-            if (now <= auctionStart + biddingTime)
-                throw; // auction did not yet end
-            if (ended)
-                throw; // this function has already been called
+            require(now >= auctionStart + biddingTime); // auction did not yet end
```

Comment:
I really don't like to add parentheses around all operations. This is not javascript, it would make no sense to have `(now >= auctionStart) + biddingTime`...